src/features/pageObjects/components/PageObjCopyButton.jsx

```diff
@@ -28,7 +28,7 @@ export const PageObjCopyButton = ({ elements }) => {
   return (
     <Tooltip placement="bottom" title={copyTooltipTitle}>
       <Button
-        className="jdn__locatorsList_button jdn__locatorsList_button-copy"
+        className="jdn__locatorsList_button jdn__locatorsList_button-copy jdn__pageObj_button-copy"
```

Comment:
does `jdn__locatorsList_button-copy` still make any sense?

Reply:
yes, they have another styles.
But I believe that there is the same issue with long locator name in locators list and we should revise it in future.

<img width="332" alt="Screenshot 2023-02-08 at 12 04 30" src="https://user-images.githubusercontent.com/59090607/217484177-ee3b4ca2-ae94-4078-9972-24343fbd854f.png">

Reply:
this style is fully overwritten by `.jdn__pageObject_button-menu`, that's why I suggest remove it from markup
![image](https://user-images.githubusercontent.com/82033805/217526684-0a861bb0-d191-457d-a73d-d4db4d2f257e.png)

and `jdn__locatorsList_button-copy` as well